tests/test_api/test_categories.py

```diff
@@ -1,14 +1,48 @@
 import pytest
+from time import sleep
 import json
 
 from . import TestBase
 
-from sourcemash.models import Feed
 
 def check_valid_header_type(headers):
     assert headers['Content-Type'] == 'application/json'
 
 
+class TestCategorizer(TestBase):
+    def test_get_url_categories(self, test_client, worker):
+        data = {'url':
+                "http://www.cnn.com/2014/03/27/world/ebola-virus-explainer/"}
+        r = test_client.get('/api/categorizer', query_string=data)
+        check_valid_header_type(r.headers)
+        assert r.status_code == 200
+
+        data = json.loads(r.data)
+        job_id = data['job_id']
+        assert job_id
+
+        data = {'job_id': job_id}
+        r = test_client.get('/api/categorizer/results', query_string=data)
+        resp = json.loads(r.data)
+        assert resp['ready'] == False
+
+        worker.work(burst=True)
+
+        r = test_client.get('/api/categorizer/results', query_string=data)
+        resp = json.loads(r.data)
+
+        assert {'name': "Ebola Virus Epidemic In West Africa"} in resp['categories']
```

Comment:
The worker.work line is what actually starts the job. Burst=True makes it run sequentially.